packages/vitest/src/types/reporter.ts

```diff
@@ -7,6 +7,7 @@ export interface Reporter {
   onPathsCollected?: (paths?: string[]) => Awaitable<void>
   onCollected?: (files?: File[]) => Awaitable<void>
   onFinished?: (files?: File[], errors?: unknown[]) => Awaitable<void>
+  onFinally?: () => Awaitable<void>
```

Comment:
Why do we need another hook? Why is `onFinished` not enough?

Reply:
I need to await `this.state.finishCollectingPaths()` the `onFinished` hook will go into endless loop

Reply:
Should we move `onFinished` then?

Reply:
ok